bokeh/models/widgets/markups.py

```diff
@@ -103,6 +107,12 @@ class Div(Markup):
     The default value is ``False``, meaning contents are rendered as HTML.
     """)
 
+    render_as_mathtext = Bool(False, help="""
```

Comment:
suggest framing this as `enable_mathtext` which "only applies when rendering HTML (that is, when `render_as_text` is False)"